docs/INSTALLATION.md

```diff
@@ -25,161 +46,144 @@ De opbouw van dit certificaat is als volgt:
-|    |    |
-|    |    |
-
-**_Gekoppelde bronnen: KISS-Elastic-Sync_**
-
-KISS-Elastic-Sync is het component dat zorgt dat de gekoppelde bronnen die via Elasticsearch ontsloten worden in KISS, naar de juiste Indexen worde gepushed, met de benodigde gegevens hieraan toegevoegd. Onderstaande environment variabelen gaan over de bronnen die gekoppeld zijn aan de KISS-Elastic-Sync.
-
-| Variabele | Uitleg |  
-|---|---|
-| enterprise_search_url | URL van de API waarop KISS de elastic instantie <br />kan bevragen | 
-| enterprise_search_private_api | Nodig om de Elastic API te vullen |
-| MEDEWERKER_OBJECTEN_BASE_URL | URL van de Objecten API waar de medewerkers <br />gevonden kunnen worden |
-| MEDEWERKER_OBJECTEN_TOKEN |  Token van de Objecten API waar de medewerkers <br />gevonden kunnen worden<br/>**LET OP: niet te gebruiken** in combinatie met MEDEWERKER_OBJECTEN_CLIENT_SECRET en MEDEWERKER_OBJECTEN_CLIENT_ID  |
-| MEDEWERKER_OBJECTEN_CLIENT_ID |  Client ID van de Objecten API waar de medewerkers <br />gevonden kunnen worden<br/>**LET OP: niet te gebruiken** in combinatie met MEDEWERKER_OBJECTEN_TOKEN |
-| MEDEWERKER_OBJECTEN_CLIENT_SECRET |  Client Secret van de Objecten API waar de medewerkers <br />gevonden kunnen worden<br/>**LET OP: niet te gebruiken** in combinatie met MEDEWERKER_OBJECTEN_TOKEN |
-| MEDEWERKER_OBJECTTYPES_BASE_URL | URL van de Objecttype API waar het objecttype <br />'Medewerker' gevonden kan worden |
-| MEDEWERKER_OBJECTTYPES_TOKEN | Token van de Objecttype API waar het objecttype <br />'Medewerker' gevonden kan worden |
-| VAC_OBJECTEN_BASE_URL | URL van de Objecten API waar de VAC's gevonden <br />kunnen worden |
-| VAC_OBJECTEN_TOKEN |  Token van de Objecten API waar de VAC's gevonden <br />kunnen worden |
-| VAC_OBJECTTYPES_BASE_URL | URL van de Objecttype API waar het objecttype 'VAC' <br />gevonden kan worden |
-| VAC_OBJECTTYPES_TOKEN | Token van de Objecttype API waar het objecttype 'VAC' <br />gevonden kan worden |
-| SDG_BASE_URL  | URL van de API waarmee Kennisartikelen <br />(PDC-producten) kunnen worden opgehaald door KISS |
-| SDG_API_KEY  | URL van de API voor Kennisartikelen  |
-
-
-### Installatie
-Als het kubernetes cluster is ingericht, kunnen we de onderdelen van KISS gaan installeren.
-
-#### Placeholders
-De yaml-voorbeeldbestanden staan in [https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/blob/main/helm/kiss-frontend/kiss.template.yaml](https://github.com/Klantinteractie-Servicesysteem/KISS-frontend/blob/main/helm/kiss-frontend/kiss.template.yaml)
-
-#### Uitvoeren
-De installatie kan worden uitgevoerd middels onderstaande powershell script. Ook zonder powershell zijn de commands in het script handmatig uit te voeren.
+## Configuratie: Environment Variabelen
+
+Hieronder staan de benodigde environment variabelen per onderdeel van KISS.

```

Comment:
Lekker korte en compacte intro op dit onderdeel :-)
Een vraag: de toevoeging over default waarden in het installatiescript zijn weg. Klopt het dat die dan overbodig waren?